Cargo.toml

```diff
@@ -5,6 +5,12 @@ name = "{{project-name}}"
 edition = "2018"
 version = "0.1.0"
 
+[lib]
+test = false
+
+[workspace]
+members = ["testsuite"]
```

Comment:
so tests are in their crate to about writing `[[bin]].harness = false` and what not in the main Cargo.toml?

Reply:
Yeah